Dockerfile

```diff
@@ -1,22 +1,12 @@
-# Use distroless for ca certs.
-FROM gcr.io/distroless/static AS distroless
+FROM gcr.io/distroless/static
```

Comment:
I just decided to use distroless/static directly, since the binary is pretty fat and might need the other things distroless provides.

Reply:
Would that be the recommendation going forward? cgr.dev/chainguard/static:latest chainguard alternative claims zero CVEs